Makefile

```diff
@@ -808,6 +809,20 @@ check-staticcheck: \
 		-tags $(GO_TAGS_EBPF) \
 		./...
 
+.PHONY: check-err
+check-err: \
+	.checkver_$(CMD_GO) \
+	tracee-ebpf \
```

Comment:
shouldn't we use tracee instead of tracee-ebpf?

Reply:
Of course, but as you can see all other checks are still calling tracee-ebpf. I' assuming that when we get rid of tracee-ebpf and tracee-rules we will change all calls to them.